docs/running-on-mesos.md

```diff
@@ -226,6 +226,20 @@ See the [configuration page](configuration.html) for information on Spark config
     The final total amount of memory allocated is the maximum value between executor memory plus memoryOverhead, and overhead fraction (1.07) plus the executor memory.
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.coarse.cores.max</code></td>
+  <td>Int.MaxValue</td>
+  <td>
+    The maximum amount of cores the coarse grained mesos scheduler will allocate per executor. This only applies for coarse grained mode.
```

Comment:
The maximum number of cores to use on each Spark executor. Coarse-grained mode only.